src/xcode/Dashcore.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,4350 @@
+		FBEFB4E51E6553CC000C6EB8 /* connect4_16.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = connect4_16.png; sourceTree = "<group>"; };
+		FBEFB4E61E6553CC000C6EB8 /* debugwindow.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = debugwindow.png; sourceTree = "<group>"; };
+		FBEFB4E71E6553CC000C6EB8 /* drkblue_address-book.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = "drkblue_address-book.png"; sourceTree = "<group>"; };
+		FBEFB4E81E6553CC000C6EB8 /* drkblue_editcopy.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = drkblue_editcopy.png; sourceTree = "<group>"; };
+		FBEFB4E91E6553CC000C6EB8 /* drkblue_editpaste.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = drkblue_editpaste.png; sourceTree = "<group>"; };
+		FBEFB4EA1E6553CC000C6EB8 /* drkblue_remove.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = drkblue_remove.png; sourceTree = "<group>"; };
+		FBEFB4EB1E6553CC000C6EB8 /* edit.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = edit.png; sourceTree = "<group>"; };
+		FBEFB4EC1E6553CC000C6EB8 /* editcopy.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = editcopy.png; sourceTree = "<group>"; };
+		FBEFB4ED1E6553CC000C6EB8 /* editpaste.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = editpaste.png; sourceTree = "<group>"; };
+		FBEFB4EE1E6553CC000C6EB8 /* export.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = export.png; sourceTree = "<group>"; };
+		FBEFB4EF1E6553CC000C6EB8 /* eye.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = eye.png; sourceTree = "<group>"; };
+		FBEFB4F01E6553CC000C6EB8 /* eye_minus.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = eye_minus.png; sourceTree = "<group>"; };
+		FBEFB4F11E6553CC000C6EB8 /* eye_plus.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = eye_plus.png; sourceTree = "<group>"; };
+		FBEFB4F21E6553CC000C6EB8 /* filesave.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = filesave.png; sourceTree = "<group>"; };
+		FBEFB4F31E6553CC000C6EB8 /* history.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = history.png; sourceTree = "<group>"; };
+		FBEFB4F41E6553CC000C6EB8 /* key.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = key.png; sourceTree = "<group>"; };
+		FBEFB4F51E6553CC000C6EB8 /* lock_closed.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = lock_closed.png; sourceTree = "<group>"; };
+		FBEFB4F61E6553CC000C6EB8 /* lock_open.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = lock_open.png; sourceTree = "<group>"; };
+		FBEFB4F71E6553CC000C6EB8 /* masternodes.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; path = 
```

Comment:
This file doesn't exist anymore in 12.2. There are probably other files that has to be included (for example main.cpp was split into validation.cpp and net_processing etc etc).